app/views/posts/_post.html.erb

```diff
@@ -1,18 +1,16 @@
-<div class="post-main-wrap">
-<div class="post-wrap">
+<div class="card">
  <h2>
-    <span>Post #<%= post.id  %></span>
+    <span>Post #<%=  %></span>
     <span>
       <%= post.title %>
     </span>
   </h2>
    <p>
     <%= post.text %>
   </p>
-  </div>
-  <div class="counter">
+  <div class="comment">
   <small>
-    <span>Comment: <%= post.comments_counter %>, </span>
+    <span>Comment: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></span>
   </small>
   </div>
 </div>
```

Comment:
- [ ] I am not able to see any kind of form to create posts and comments. Also, I'm not able to like the posts. Please work on that.

![Screenshot from 2022-07-17 02-18-08](https://user-images.githubusercontent.com/26767383/179371558-87047f52-99c5-4f27-a9e4-1eff107e9b0a.png)